inc/composer/class-command.php

```diff
@@ -320,18 +335,135 @@ protected function logs( InputInterface $input, OutputInterface $output ) {
 	protected function shell( InputInterface $input, OutputInterface $output ) {
 		$columns = exec( 'tput cols' );
 		$lines = exec( 'tput lines' );
+		$command_prefix = $this->get_base_command_prefix();
 		passthru( sprintf(
-			'cd %s; VOLUME=%s COMPOSE_PROJECT_NAME=%s docker-compose exec -e COLUMNS=%d -e LINES=%d php /bin/bash',
-			'vendor/altis/local-server/docker',
-			escapeshellarg( getcwd() ),
-			$this->get_project_subdomain(),
+			"$command_prefix docker-compose exec -e COLUMNS=%d -e LINES=%d php /bin/bash",
 			$columns,
 			$lines
 		), $return_val );
 
 		return $return_val;
 	}
 
+	protected function db( InputInterface $input, OutputInterface $output ) {
+		$db = $input->getArgument( 'options' )[0] ?? '';
+		$columns = exec( 'tput cols' );
+		$lines = exec( 'tput lines' );
+
+		$base_command_prefix = $this->get_base_command_prefix();
+
+		$base_command = sprintf(
+			"$base_command_prefix docker-compose exec -e COLUMNS=%d -e LINES=%d db",
+			$columns,
+			$lines
+		);
+
+		$return_val = 0;
+
+		switch ( $db ) {
+			case 'info':
+				$connection_data = $this->get_db_connection_data();
+
+				$db_info = <<<EOT
+<info>Root password</info>:  ${connection_data['MYSQL_ROOT_PASSWORD']}
+
+<info>Database</info>:       ${connection_data['MYSQL_DATABASE']}
+<info>User</info>:           ${connection_data['MYSQL_USER']}
+<info>Password</info>:       ${connection_data['MYSQL_PASSWORD']}
+
+<comment>Version</comment>:        ${connection_data['MYSQL_VERSION']}
+<comment>Host</comment>:          ${connection_data['HOST']}
+<comment>Port</comment>:          ${connection_data['PORT']}
+
+EOT;
+				$output->write( $db_info );
+				break;
+			case 'spf':
+				$connection_data = $this->get_db_connection_data();
+				$rpf_file_contents = file_get_contents( getcwd() . '/vendor/altis/local-server/templates/sequel.xml' );
```

Comment:
Any reason not to use `dirname( __DIR__, 2 )` or other relative path method? Whenever possible it's good to avoid writing out the full `vendor/namespace/package` paths because it's possible for projects to override the vendor directory location from the root `composer.json`

Reply:
Where is the "rpf" name coming from? Should that be "spf"?

Reply:
There was no reason. I tried first a relative way but saw that `getcwd` was used often so I thought I was being consistent.